src/ansys/fluent/core/quantity.py

```diff
@@ -693,7 +693,7 @@ def __sub__(self, other):
             temp_value = self._si_value - other._si_value
```

Comment:
Maybe you can use `validate_matching_dimensions` for `__add__` and `__sub__` as well.

Reply:
Right. Done. Thank you.